docs/reference/snapshot-restore/apis/repo-analysis-api.asciidoc

```diff
@@ -302,19 +312,31 @@ in the test.
 =====
 `count`::
 (integer)
-The numer of write operations performed in the test.
+The number of write operations performed in the test.
 
-`total_bytes`::
-(integer)
+`total_size`::
+(string)
+The total size of all the blobs written in the test.
```

Comment:
I wonder if we should be explicit about the difference between the human readable field and the regular field. Is this something we do in other APIs?

Reply:
IMO the `(string)` vs `(long)` and the fact that we say one is in nanoseconds and don't say anything about the other is enough. I considered linking the [time units docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/common-options.html#time-units) but technically these things aren't time units since time units don't support decimals.